Extension/bin/msvc.json

```diff
@@ -1,11 +1,7 @@
   {
     "defaults": [
       "cpfe",
```

Comment:
--no_warnings is added if the warningSquiggles is "Disabled". The other removed flags are added only for C++ translation units.